doc/overview.md

```diff
@@ -0,0 +1,97 @@
+# The Projects
+The projects in order of dependency (with most abstract at top) are:
+1. Mantle - libraries of useful circuits built in Magma
+1. Magma - A Python API for creating circuits
+1. PyCoreIR - A set of Python bindings for converting Magma into CoreIR
+1. CoreIR - A C++ library for manipulating the CoreIR hardware IR (like LLVM for hardware)
+
+__NOTE: Where does loam fit in here?__
```

Comment:
Loam extends magma with abstractions for physical FPGAs, Peripherals, Parts and Boards, so a higher level of "hardware abstraction" above mantle.